frontend/amundsen_application/static/js/components/Badges/BadgeList/index.tsx

```diff
@@ -82,7 +82,11 @@ export default class BadgeList extends React.Component<BadgeListProps> {
             badgeConfig = getBadgeConfig(badge.badge_name);
           }
 
-          if (badge.badge_name && badge.category === COLUMN_BADGE_CATEGORY) {
+          if (
+            badge.badge_name &&
+            badge.category === COLUMN_BADGE_CATEGORY &&
+            !hideNonClickableBadges()
```

Comment:
Perhaps you could rename this function to something like "isClickable()" for less cognitive complexity, rather than a double-negative which may be a bit harder to read ...

Reply:
I've actually intentionally named this function `hideNonClickableBadges` to match with the custom property we add, which for me is more explicit than `isClickable`

Reply:
Got your point, but actually I don't think current naming even has double negatives; `notHideNonClickableBadges` could be an example but current naming is `hideNonClickableBadges`, so I don't see an issue

Reply:
I think it does; For example, ! Non Clickable = Clickable ... but I guess it's a matter of perspective